src/controllers/tasks/statement.of.capital.controller.ts

```diff
@@ -55,4 +60,27 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
   }
 };
 
+const sendUpdate = async (transactionId: string, submissionId: string, req: Request) => {
+  const statementOfCapital: StatementOfCapital = req.sessionCookie["statementOfCapital"];
```

Comment:
could you add a 'sessionCookieConstants' exported object in constants.ts and add the "statementOfCapital" to it and use it on line 64 then. That way, we have a central place to store the session cookie key names.